README.md

```diff
@@ -79,15 +79,24 @@ We will provide:
 * Vignettes on how to create ADSL, BDS and OCCDS datasets, including example scripts
 * Vignettes for ADaM dataset specific functionality (i.e. dictionary coding, date imputation, SMQs ...)
 
-## Types of Packages
+## The {admiral} Family of Packages
 
-There will be 3 foreseeable types of `{admiral}` packages:
+There are three types of packages in the `{admiral}` family:
 
 * Core package---one package containing all core functions required to create ADaMs, usable by any company (i.e. general derivations, utility functions and checks for ADSL, OCCDS and BDS)
 * TA (Therapeutic Area) package extensions---one package per TA with functions that are
   specific to algorithms and requirements for that particular TA (e.g. [`{admiralonco}`](https://pharmaverse.github.io/admiralonco/))
 * Company package extensions---specific needs and plug-ins for the company, such as access to metadata
   (e.g. `{admiralroche}` or `{admiralgsk}`)
+  
+## Related Data Packages
```

Comment:
Should we talk about `xportr`, `metacore` and `metatools` for related ADaM needs??  Maybe `sdtmchecks` or `oak` could get a brief shout out??

Reply:
Added mention of `xportr`, `metacore` and `metatools` - thanks for the suggestion. As this is in a "related ADaM needs" section I omitted mentioning `sdtmchecks` and `oak` which are more focused on SDTM.